### PR TITLE
[Merge After #65] Get next student via mention

### LINKED ISSUE #61: Automatically taken to next student
As an instructor, I would like to (ideally) automatically be taken to a conversation with the next student in line after the conversation with the previous student in line has been resolved.

### PR DESCRIPTION
**Pull request by rorochaudhary**
Continues from #65 and resolves #61.

Since it has been found that avenues to create group conversations between Students and Instructors are not implemented or not available due to lack of institutional admin privileges, a consolation is to mention/tag the new student from the queue to bring up a notification and inform them to get into contact with the instructor (who is also mentioned/tagged).

Screenshot of functionality:
image

### CODE REVIEW

**Comment by Jaqito** [comment]
Don't have time to review to full thing right now but next student to be helped should be mapped to the getNameOfTeamMembers api call that's in the master branch. Make the output "Rohit is the next student to be helper" instead of showing the userIds etc.

**Comment by rorochaudhary** [comment]
> Don't have time to review to full thing right now but next student to be helped should be mapped to the getNameOfTeamMembers api call that's in the master branch. Make the output "Rohit is the next student to be helper" instead of showing the userIds etc.

Would it be better to use `TeamsInfo.getMember`? It directly gets the member of interest and avoids the extra work needed to find the single member from the list of members returned. Additionally, the bot currently only allows for a single member at a time to be helped.

Hm, since the bot is currently allowing Students to use ideally Instructor-only commands and vice-versa, would it be better to err on the side of explicitness and tag bot the Instructor and the Student to let them both know to get connected? For the ids being shown are you referring to the stuff titled "Next student to be helped"?

**Comment by Jaqito** [comment]
Ah I didn't even notice that Teams call you're right that that would be way easier to use. @rorochaudhary 

I'm saying the output for the bot should after "get next student"  should be something like Next student to be helper is ${studentName}!  

There shouldn't be any references to ids as it's not pretty for the users to see.

File: bot/teamsBot.ts [hunk 1]
```diff
@@ -160,6 +172,76 @@ export class TeamsBot extends TeamsActivityHandler {
...
+            nextInLine.setResolvedState(StudentStatus.Conversing);
+            const updateResult = await updateQueueEntryResolved(
+              this.dbConnection,
+              nextInLine.id,
+              nextInLine.resolved
+            );
+            console.log(`Updated: ${updateResult}`);
+            await context.sendActivity(
+              `Next student to be helped:${nextInLine.toString()}`
+            );
```
Comment (Jaqito) [inline bot/teamsBot.ts]: Only one change I see. Can you delete this line?

**Review by Jaqito — CHANGES_REQUESTED** [review]
Just one change needed. Nice work!

File: bot/teamsBot.ts (hunk 1 above)
Comment (rorochaudhary) [reply bot/teamsBot.ts]: How does this look? Explicit response to the command itself followed by tagging the user. 
image

**Comment by Jaqito** [comment]
Perfect ya mate

**Review by Jaqito — APPROVED** [review]
Nice work!

**Pull request merged**